jupyterhub_chameleon/spawner.py

```diff
@@ -181,14 +181,13 @@ def get_env(self):
         # Add parameters for experiment import
         artifact = self.get_artifact()
         if artifact:
-            deposition_url = artifact.deposition_url()
-            extra_env["ARTIFACT_DEPOSITION_URL"] = deposition_url
+            extra_env["ARTIFACT_CONTENTS_URL"] = artifact.contents_url
```

Comment:
Why does this require using`contents_url` instead of `deposition_url()` now? Where is `contents_url` created?

Reply:
Jason said he wanted to get rid of the "deposition" language in these projects. `contents_url` is derived in Portal, and then sent to JupyterHub as a URL parameter. JupyterHub derives it in `Artifact.from_query`. We don't want JupyterHub to have to generate the content URL on its own, as it would require us to port Trovi code from Lab to Hub. A much easier solution was to just have Portal figure out that URL and send it on.